### PR TITLE
Integration specs pull request

### PR DESCRIPTION
**Pull request by AleWaweru**
- [ ] Solved the n+1 issue
- [ ] Integration specs using  Capybara for:
- User index page
- User show page
- User post index page
- Post show page

- Before
Screenshot (82)

-After

Screenshot (83)

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @AleWaweru @bronnie313 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- This is a pair-programming milestone. You ought to make contributions to the code but @bronnie313 only made changes to the README file which doesn't count as collaboration. Kindly collaborate properly ✔️  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: spec/features/user_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.feature 'User Index', type: :feature do
```
Comment (Olamarx) [inline spec/features/user_index_spec.rb:3]: - [ ] While you tested most of the required aspect, you probably forgot the pointed one in the image
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme file professional ✔️
- All test cases passed ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
## STATUS: APPROVED 👍🎯  
Hello @AleWaweru  👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ The test was added as required 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/features/user_index_spec.rb (hunk 1 above)
Comment (AleWaweru) [reply spec/features/user_index_spec.rb:3]: done

**Pull request merged**